lib/rbs/json_schema/generator.rb

```diff
@@ -0,0 +1,453 @@
+require "json"
+require "uri"
+require "rbs/json_schema/validation_error"
+
+module RBS
+  module JSONSchema
+    class Generator
+      attr_reader :options
+      attr_reader :stdout
+      attr_reader :stderr
+      attr_reader :decls
+      attr_reader :generated_schemas
+
+      # SCHEMA DEFINITIONS required for generating RBS from JSON schema
+      SCHEMA_DEFINITIONS = %w(definitions properties type enum const allOf oneOf items $ref $schema title description).freeze
```

Comment:
I don't think declarating this constant makes much sense. Writing literals directly is better.